var/spack/repos/builtin/packages/lsof/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from os import system
+
+
+class Lsof(Package):
+    """Lsof displays information about files open to Unix processes."""
+
+    homepage = "https://people.freebsd.org/~abe/"
+    url      = "https://www.mirrorservice.org/sites/lsof.itap.purdue.edu/pub/tools/unix/lsof/lsof_4.89.tar.gz"
+
+    version('4.89', '8afbaff3ee308edc130bdc5df0801c8f')
+
+    def install(self, spec, prefix):
+        system("tar xf lsof_{0}_src.tar && cd lsof_{0}_src"
+               " && ./Configure -n linux && make".format(self.version))
```

Comment:
A more Spack-esque way to do this would be:
```python
tar = which('tar')
tar('xf', 'lsof_{0}_src.tar'.format(self.version))

with working_dir('lsof_{0}_src'.format(self.version)):
    configure = Executable('./Configure')
    configure('-n', 'linux')

    make()
```